sunspec/core/modbus/mbmap.py

```diff
@@ -22,12 +22,17 @@
 """
 
 import struct
+import sys
 
 try:
     import xml.etree.ElementTree as ET
 except:
     import elementtree.ElementTree as ET
 
+# Python 3 compatibility for long()
+if sys.version_info > (3,):
+    long = int
```

Comment:
I thought that int elevated to long as necessary in py2 so no need for using long at all.